src/Pages/Repositories/ContentListTable/components/UploadContent/compontents/FileUploader.tsx

```diff
@@ -0,0 +1,339 @@
+import { useEffect, useMemo, useState } from 'react';
```

Comment:
must have missed this in the upload PR, typo in this directory name

Reply:
Compontents....